assume/common/grid_utils.py

```diff
@@ -195,31 +195,34 @@ def add_loads(
         )
 
 
-def add_redispatch_loads(
+def add_fix_units(
```

Comment:
The doc strings should be adjusted for this method as well - though "fix units" does not provide meaning to me